src/pages/components/tabs/style.mdx

```diff
@@ -105,6 +105,18 @@ structure, and size.
   Examples of enabled, hover, and active states for scrollable line tabs.
 </Caption>
 
+{/* prettier-ignore-start */}
+
+{/* 
+TODO: 
+1. Are all of the 'dismissible' strings supposed to be 'dismissable' instead?
```

Comment:
It should be spelled correctly, IMO. Not sure of the original intent (I'm guessing just a typo), but it's either gone unnoticed or just easier to leave it as-is. This would be good to fix for v12, I think. We could introduce the new (correctly spelled) one now with a deprecation notice for the old one.

Unless there's context I'm missing, @tay1orjones?

Reply:
We have been calling this `dismissible`, and not dismissable across the board for code and design. I would like to get input from the IBM Style council on what the correct spelling of this should be, because I think it could technically be either of these.

Reply:
Should I expect a reply here when you receive feedback or should I wait for it elsewhere? Not sure if an internal Slack discussion was started.